src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Constants/XacmlRequestAttribute.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Altinn.Platform.Authorization.Constants
+{
+    /// <summary>
+    /// Attribute representations in XACML
+    /// </summary>
+    public static class XacmlRequestAttribute
+    {
+        /// <summary>
+        /// xacml string that represents org
+        /// </summary>
+        public const string OrgAttribute = "urn:altinn:org";
+
+        /// <summary>
+        /// xacml string that represents app
+        /// </summary>
+        public const string AppAttribute = "urn:altinn:app";
+
+        /// <summary>
+        /// xacml string that represents isntance
+        /// </summary>
+        public const string InstanceAttribute = "urn:altinn:instance-id";
+
+        /// <summary>
+        /// xacml string that represents task
+        /// </summary>
+        public const string TaskAttribute = "urn:altinn:task";
+
+        /// <summary>
+        /// xacml string that represents party
+        /// </summary>
+        public const string PartyAttribute = "urn:altinn:partyid";
```

Comment:
is there a reason we don't use party-id as we have for instance and user here ?

Reply:
nope, i just reused the attributes that was set by Rune. I shall check with him if a "-" must be introduced

Reply:
After discussions with @TheTechArch , we agreed to remove the "-" in all the attributes

Reply:
It's fixed in the context handler implementation and tests but must also be changed in the decision point implementation and tests. Created issue [https://github.com/Altinn/altinn-studio/issues/2922](url)